.dockerignore

```diff
@@ -1,2 +1,4 @@
 docker-machine*
-.git
```

Comment:
It's 11MB currently.
The total size of the folder is about 50MB.
So, it's a 20% increase on the size of the build context.

wdyt?